mafia_bot.py

```diff
@@ -60,7 +53,10 @@ def start_room(driver,game_name='',listed=0):
     submit.click()
     time.sleep(0.5)
     chat(driver,'Bot is ready!')
-    driver.find_elements_by_class_name('button')[0].click()
+    try:
+        driver.find_elements_by_class_name('button')[0].click()
+    except:
+        pass

```

Comment:
add a comment as to why this skip is okay/expected

Reply:
ah the only except case is when the bot creates a lobby and by default with the way the setup_room function works, it always clicks on the player Up button to become a spectator.  however, the button is non-interactable when the lobby is a closed setup (since host can't player up) and runs into that error unless you do a simple try catch,  nothing has to be handled im pretty sure.

Reply:
yep, just add some succinct description in the code as a comment!